src/ecSubst.ml

```diff
@@ -426,8 +426,7 @@ and subst_instr (s : subst) (i : instr) : instr =
 
      i_match (e, bs)
 
-  | Sassert e ->
-     i_assert (subst_expr s e)
+  | Sraise e -> i_raise e
```

Comment:
patch need to be substituted.
But if you use expression in Sraise the problem will be solved